Formula/luit.rb

```diff
@@ -13,11 +13,13 @@ class Luit < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}", "--with-encodings-dir=auto"
     system "make", "install"
   end
 
   test do
-    system "#{bin}/luit", "-list"
+    require "pty"
+    PTY.spawn(bin/"luit", "-encoding", "GBK", "ls") do |r, _w, _pid|
```

Comment:
What is the test trying to do?

Reply:
Run binary with `-encoding GBK` will call `encodings-dir`, but this require `tcgetattr` in `pty`, I found solution here https://github.com/Homebrew/homebrew-core/search?q=tcgetattr&type=Issues